rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -126,10 +126,13 @@ def test_ppo_compilation_and_schedule_mixins(self):
                     config.training(model=get_model_config(fw, lstm=lstm))
 
                     algo = config.build(env=env)
+                    learner = algo.learner_group._learner
```

Comment:
I'm not sure whether this is a good idea. What if the LearnerGroup doesn't have a local Learner? Here in this test, we know for sure that we are dealing with a local Learner inside the LearnerGroup. Doing this here is just for a very specific inspection job, nothing else. I don't really want users to poke around the internals like this.

Alternatively, we need more discussion on whether we would like to add utilities like `foreach_learner`, etc.. as we already have this for `WorkerSet` on the sampling side.

Adding a TODO.

Reply:
done

Reply:
yeah. the only downside I see is that this test written this way has to be upgraded if we upgrade the internal implementation of `LearnerGroup`s. For example, let's say tomorrow, instead of `_learner` attribute I use `_local_learner` attribute. This test will break then, while the testing strategy is still valid. Idk, sometimes I think we should never write a test for private behaviors of modules, we should test only the public behaviors.